### ISSUE TITLE
Empty link name at llvmdeps.rs

### ISSUE THREAD

**Issue by mopp** [rust-lang/rust#28671]
Hi everyone.

I tried to build rust from github source ().
But, I faced below error.

configure option is here.
```shell
 ./configure --target=i686-unknown-linux-gnu,x86_64-unknown-linux-gnu --enable-clang
```
Then 
```shell
rustc: x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_llvm
/home/mopp/Workspace/repo/rust/x86_64-unknown-linux-gnu/rt/llvmdeps.rs:77:38: 77:54 error: #[link(name = "")] given with empty name
/home/mopp/Workspace/repo/rust/x86_64-unknown-linux-gnu/rt/llvmdeps.rs:77 #[cfg_attr(not(target_env = "msvc"), link(name = ""))]
                                                                                                               ^~~~~~~~~~~~~~~~
note: in expansion of include!
src/librustc_llvm/lib.rs:2336:5: 2336:47 note: expansion site
error: aborting due to previous error
/home/mopp/repo/rust/mk/target.mk:164: recipe for target 'x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib/stamp.rustc_llvm' failed
make: *** [x86_64-unknown-linux-gnu/stage0/lib/rustlib/x86_64-unknown-linux-gnu/lib/stamp.rustc_llvm] Error 101
```

I'm very sorry if this error is reported already

**Comment by sfackler** [comment]
I've run into that recently as well, but deleting llvmdeps.rs and rebuilding seems to fix everything. Not really sure what's causing it though.

**Comment by mopp** [comment]
@sfackler thank you for your comment !
I did same things and succeed in building.

**Comment by wthrowe** [comment]
This is likely an effect of #28614.

**Comment by mopp** [comment]
Thanks a lot.
It is better.
This issue will be close
